test/micro/org/openjdk/bench/java/lang/ref/ReferenceClear.java

```diff
@@ -0,0 +1,82 @@
+//
+// * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Drive-by comment: The `// *` format looks weird.

Reply:
Actually, this constellation of single-line comments should be replaced with a multi-line comment block. The fix is now in my (unpublished) patch queue, thanks.

Reply:
Yes, that's what I meant. Thanks.